testsuite/setup.py

```diff
@@ -213,7 +213,7 @@ def dynamic_author_list():
           long_description=LONG_DESCRIPTION,
           install_requires=[
               'MDAnalysis=={0!s}'.format(RELEASE),  # same as this release!
-              'nose>=1.3.7',
+              'numpy>=1.10.4',
```

Comment:
Please remove this line when you do the rebase. It's not there in develop right now.